### PR TITLE
Export blurbs to yml

### PR DESCRIPTION
**Pull request by chrishunt**
Add export to Copycopter cache with accompanying rake task and readme.

### CODE REVIEW

**Comment by croaky** [comment]
Thanks, Chris. We'll have to take a closer look at the commit later but at first glance, it looks pretty good. Out of curiosity, what are you using the exported YAML for at this point? Would love to know from a product perspective.

**Comment by chrishunt** [comment]
We have a ton of copy in our app and it's legally binding. Copycopter looks awesome, but in order to get product to approve of something like this we need to be able to export the blurbs for checking into source control. This is to cover our butts in the (unlikely) event that the Copycopter services disappears.

**Comment by gabebw** [comment]
I'd like to merge this, but I get test failures on multiple ruby versions: https://gist.github.com/1836737

**Comment by chrishunt** [comment]
Thanks Gabe. I only only tested on 1.9.3. What ruby versions do you test against? I'll get things working on 1.8.9 and 1.9.2 today and update.

**Comment by gabebw** [comment]
Hash ordering is not guaranteed in 1.8.7, so the expected YAML sometime matches and sometimes doesn't. One way to get around this is to do this:

``` ruby
      it "returns blurbs as yaml" do                                                                                                                                                                                                                                                  
        exported = YAML.load(@cache.export)                                                                                                                                                                                                                                           
        exported['en']['test']['key'].should == 'en test value'                                                                                                                                                                                                                       
        exported['en']['test']['other_key'].should == 'en other test value'                                                                                                                                                                                                           
        exported['fr']['test']['key'].should == 'fr test value'                                                                                                                                                                                                                       
      end  
```

File: spec/copycopter_client/cache_spec.rb [hunk 1]
```diff
@@ -205,5 +205,78 @@ def violated(failure)
...
+      end
+    end
+
+    context "with conflicting blurb keys" do
+      let(:save_blurbs) do
+        client['en.test.key'] = 'test value'
+        client['en.test']     = 'other test value'
+      end
+
+      it "retains the new key" do
```
Comment (gabebw) [inline spec/copycopter_client/cache_spec.rb:267]: This sometimes fails and sometimes passes, depending on which key is processed first in the `@blurbs.each` block. If `en.test.key` is saved first, it passes. If `en.test` is processed first, then it tries to use `key` as an index into "other test value" and fails.

**Comment by gabebw** [comment]
We test against 1.8.7 and 1.9.2.

**Comment by chrishunt** [comment]
Gabe, thanks for the suggestion on testing the YAML output. That's much cleaner. Tests are passing for me now on `1.8.7`, `1.9.2`, and `1.9.3`.

**Pull request merged**